librecad/src/lib/engine/rs_ellipse.cpp

```diff
@@ -1324,7 +1324,7 @@ const RS_EllipseData& RS_Ellipse::getData() const
  * x *= kx
  * y *= ky
  * find the maximum and minimum of x^2 + y^2,
- */
```

Comment:
I just wanted to comment away the former scale() member function. This LibreCAD code compiled on my machine before I went on to make a pull request.